code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -247,10 +255,6 @@
 	if(method == INGEST)
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
-
-			if(volume < 10)
-				to_chat(M, "<span class='danger'>The greenish acidic substance stings you, but isn't concentrated enough to harm you!</span>")
```

Comment:
You can already taste the acid with a massive taste message, so two different outputs on the taste seem redundant to me.